ts/src/base/Exchange.ts

```diff
@@ -3765,12 +3765,19 @@ export default class Exchange {
             precision = this.safeValue (networkItem, 'precision', precision);
         }
         if (precision === undefined) {
-            return fee;
+            return this.forceString (fee);
```

Comment:
this change is in base. it affects **only few exchanegs** where currency precisions are not available from `fetchCurrencies`.
for those exchanges, we need to return `string` , because 2 lines down here, in `else` clause we return `string` (as string is returned from `decimalToPrecision` ). 
so, currently we have a bug as we are returning different types from this `currencyToPrecisdion` function atm (you can see that by searching `this.currencyToPrecision` across lib to see how we are passing `numeric` type in second argument, thus returning that value directly would mean returning numeric number)

Reply:
@ttodua u sure this won't break things? Most of the time we're adding this result directly to the request and the exchange might not accept strings in the request

Reply:
another alternative is, to wait for `createOrder` tests to be merged into tests, and we could then test :))

Reply:
@carlosmiei I've updated top-post, please re-read, i think it clarifies more.

Reply:
@ttodua sorry I'm not 100% clear yet, I think it is easier with an example: 

Right now poloniex does this inside `transfer`:

```Javascript
        amount = this.currencyToPrecision (code, amount);
        const request = {
            'amount': amount,
        };
```
but this PR might be changing the amount type right? because now we're forcing strings.

My question is if we might be breaking things by forcing a string without checking if the exchange accepts it or not

Reply:
yes, poloniex is among those 6 exchanges that would be affected. I can check those exchanges one by one. without testing in real, it's impossible to say in advance, as we don't have `transfer, borrowMargin...etc` tests automated

Reply:
I've tested poloniex, bitget, whitebit, bitmart... and they worked with strings. so, I don't see any much room that other exchanges failed. when i search for `currencyToPrecision` across lib, I don't see any threat. can you confirm?

Reply:
@carlosmiei wdyt?

Reply:
@ttodua ok if you tested and had no impact I think we can merge, will try to run some tests on my side as well